tests/conftest.py

```diff
@@ -275,15 +271,15 @@ def tb():
 
 
 @pytest.fixture
-def create_tx(b, user_pk):
+def create_tx(alice, user_pk):
     from bigchaindb.models import Transaction
     name = f'I am created by the create_tx fixture. My random identifier is {random.random()}.'
-    return Transaction.create([b.me], [([user_pk], 1)], asset={'name': name})
+    return Transaction.create([alice_pubkey(alice)], [([user_pk], 1)], asset={'name': name})
```

Comment:
`alice.public_key` ?

Reply:
Can be accessed using the getter. I can change if you prefer that.